.gitignore

```diff
@@ -31,3 +31,5 @@ tmp/*
 .netlify
 
 scripts/yarn.lock
+
+.history
```

Comment:
https://stackoverflow.com/questions/60687569/should-one-track-files-in-ones-history-folder-in-git

After searching Google, the `.history` folder is created by user, our repo doesn't contain this folder.

Reply:
I just ignored all history in Git.